core-bundle/contao/forms/FormAltcha.php

```diff
@@ -73,6 +73,8 @@ public function parse($arrAttributes = null): string
 			return $objTemplate->parse();
 		}
 
+		$this->isSecure = $request->isSecure();
```

Comment:
Should we check for `127.0.0.1`, `localhost` and `*.localhost` here too? Otherwise many local dev setups would not be able to test the captcha even though it would technically work on localhost.

Reply:
Changed in 96cdebdff6935558780661e8ff4ee60af07c384c. I‘ve also added `::1`.

Reply:
The IP does not matter, only the domains `localhost` and `*.localhost` are supported, are they not?

Reply:
No, `http://127.0.0.1:8000` is also supported and considered secure context.